projector-agent/gradle.properties

```diff
@@ -0,0 +1 @@
+version=1.0.0
```

Comment:
Can we use the same version for the projector-agent as for projector-plugin for now?